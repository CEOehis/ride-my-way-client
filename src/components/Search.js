import React from 'react';

const Search = () => (
  <div className="instructions">
    <h2>
      Available Ride Offers
      <i className="fa fa-search " />
    </h2>
    <form className="search">
      <div className="form-group">
        <label htmlFor="origin">
          Origin
        </label>
        <input type="text" id="origin" placeholder="Where from" required />
      </div>
      <div className="form-group">
        <label htmlFor="destination">
          Destination
        </label>
        <input type="text" id="destination" placeholder="Where to" required />
      </div>
      <input className="btn btn-submit btn-orange" type="submit" value="submit" />
    </form>
    <p>Find and select your route, then sit back and relax: we will take care of the rest!</p>
  </div>
);

export default Search;
