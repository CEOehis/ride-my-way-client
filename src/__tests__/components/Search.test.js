import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Search from '../../components/Search';

describe('Form submit Search component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Search submitting />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
