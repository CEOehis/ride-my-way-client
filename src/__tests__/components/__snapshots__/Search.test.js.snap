// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form submit Search component should render correctly 1`] = `
<div
  className="instructions"
>
  <h2>
    Available Ride Offers
    <i
      className="fa fa-search "
    />
  </h2>
  <form
    className="search"
  >
    <div
      className="form-group"
    >
      <label
        htmlFor="origin"
      >
        Origin
      </label>
      <input
        id="origin"
        placeholder="Where from"
        required={true}
        type="text"
      />
    </div>
    <div
      className="form-group"
    >
      <label
        htmlFor="destination"
      >
        Destination
      </label>
      <input
        id="destination"
        placeholder="Where to"
        required={true}
        type="text"
      />
    </div>
    <input
      className="btn btn-submit btn-orange"
      type="submit"
      value="submit"
    />
  </form>
  <p>
    Find and select your route, then sit back and relax: we will take care of the rest!
  </p>
</div>
`;
